storagenode/pieces/store.go

```diff
@@ -46,42 +47,104 @@ type Info struct {
 type ExpiredInfo struct {
 	SatelliteID storj.NodeID
 	PieceID     storj.PieceID
-	PieceSize   int64
+	InPieceInfo bool
 }
 
-// DB stores meta information about a piece, the actual piece is stored in storage.Blobs
-type DB interface {
-	// Add inserts Info to the database.
-	Add(context.Context, *Info) error
+// PieceExpirationDB stores information about pieces with expiration dates.
+type PieceExpirationDB interface {
+	// GetExpired gets piece IDs that expire or have expired before the given time
+	GetExpired(ctx context.Context, expiredAt time.Time, limit int64) ([]ExpiredInfo, error)
```

Comment:
I think `expiredAt` would be more clear as `expiresBefore`

Reply:
Fair.